exampleSite/content/installation/elements/_index.en.md

```diff
@@ -9,27 +9,33 @@ keywords: [""]
 ---
 
 # Heading 1
+
 ## Heading 2
+
 ### Heading 3
+
 #### Heading 4
+
 ##### Heading 5
+
 ###### Heading 6
 
 <hr>
 
 ##### Emphasis
 
-Emphasis, aka italics, with *asterisks* or _underscores_.
+Emphasis, aka italics, with *asterisks* or *underscores*.
 
-Strong emphasis, aka bold, with **asterisks** or __underscores__.
+Strong emphasis, aka bold, with **asterisks** or **underscores**.
 
-Combined emphasis with **asterisks and _underscores_**.

```

Comment:
I am curious, why were the underscores removed here?

Reply:
@jwflory I didn't intend to make these changes. They might have happened because of some IDE extensions while saving the file. I'll remove these changes and commit. 😅

Reply:
Thanks @BeeBombshell, reviewing the updated commit now. 👍🏻